Update 03-working-with-MARC-files.md (#144)
LibraryCarpentry/lc-marcedit: episodes/03-working-with-MARC-files.md

  </td>
  <td>mrc
  </td>

>   <td>File format typically used in an  ILS or LSP. Other file extensions provided by vendors (ex. marc, dat, bin) are equivalent. Binary is a format consisting of a series of sequential bytes, each of which is eight bits in length.

Reviewer: ```suggestion
  <td>File format typically used in an ILS or LSP. Other file extensions provided by vendors (ex. marc, dat, bin) are equivalent. Binary is a format consisting of a series of sequential bytes, each of which is eight bits in length.
```

## Reset settings for the MarcEditor in Preferences

1. Click Edit → Preferences
2. Select "MarcEditor" in the Preferences window in the left-hand pane

> 3. In the right-hand pane, select Set Defaults for either font or font size

Reviewer: ```suggestion
3. In the right-hand pane, under Set MarcEditor Font Properties, select Set Defaults
```
I checked and when clicking "set defaults" both the font size and font are returned to the default settings, even if only the font or font size is highlighted in the list

4. Click Ok
  

::::::::::::::::::::::::::::::::::::::::::::::::::

:::::::::::::::::::::::::::::::::::::::::  callout

## Behavior when you reset your preferences and other popup windows

> When you click Ok to reset preferences, there is a chance that the popup window for Preferences and/or even MarcEdit main menu will close. If the MarcEdit main menu closes, sometimes it will reopen. If it does not, then click on MarcEdit again on your desktop to reopen the application. It should be noted that not all popup windows will close automatically such as Find, Replace, or the MarcValidator. However, the Select Records for Edit temporary editor will close if you click Save. These idiosyncracies for each will be covered in their upcoming episodes.

Reviewer: This doesn't happen for me when I save my preferences, only the preference pop up closes - in your testing does the MarcEdit application close for this action?. It may be important to note, though, that in order for the preference changes to display, you will need to close and reopen the MarcEditor, which would require saving your file if you haven't done so.

Reviewer: Add callout about preferences needing to be applied by restarting the MarcEditor.
